module/support/optimisation/NSGA2Optimiser/src/nsga2/Individual.cpp

```diff
@@ -0,0 +1,236 @@
+#include "Individual.hpp"
+
+#include <cmath>
+#include <fmt/format.h>
+#include <nuclear>
+
+namespace nsga2 {
+    Individual::Individual(const IndividualConfigurator& _config) : config(_config) {
```

Comment:
```suggestion
    Individual::Individual(const IndividualConfigurator& _config) : config(_config), id(0), rank(0) ... {
```
like that but finish it off